lib/registries/addon/drafts/draft/-components/managers/license-picker-manager/template.hbs

```diff
@@ -0,0 +1,18 @@
+{{yield (hash
+    fieldIsEmpty=this.fieldIsEmpty
+    emptyFieldText=(t 'registries.registration_metadata.no_license')
+    inEditMode=true
+    userCanEdit=true
+    shouldShowField=true
```

Comment:
these props are specific to `editable-field` managers hence not needed except `inEditMode`?